acaisdk/configs.ini

```diff
@@ -1,3 +1,8 @@
 [general]
-credential_endpoint = 35.237.138.197
-credential_endpoint_port = 80
+# gke
```

Comment:
Let's adjust this logic to handle two different credentials (gke as default) and then phoebe again as a configurable parameter.  My initial exploration indicates this is possible from within the configs.py file.